packages/lit-html/src/platform-support.ts

```diff
@@ -250,3 +259,8 @@ const scopeCssStore: Map<string, string[]> = new Map();
     return template;
   };
 };
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/no-misused-new
+(globalThis as any)[
```

Comment:
Is this call eliminated in the prod build? You might need:

```ts
ENABLE_SHADYDOM_NOPATCH && (globalThis....
```

Reply:
Done.